### DISCUSSION TITLE
Click element like button when it's located on the DOM

### DISCUSSION THREAD

**Discussion by Iftakharpy** [uBlockOrigin/uBlock-issues#2885] [Ideas] [answered]
![image](https://github.com/uBlockOrigin/uBlock-issues/assets/37600593/a77586ea-809c-4df0-a1b7-b097499b1bc7)
Recently I have noticed that YouTube added a popup dialogue when it fails to load ads. While I can block the element using the picker, this causes the YouTube site to not respond to mouse clicks and scrolling gets blocked as well. So I am thinking of having a feature to click elements or interact with elements with javascript or some other way when it's located would be a good feature to have.

**Comment by gothic-bum** [comment]
https://github.com/gorhill/uBlock/commit/7af88b025d

**Comment by J3ekir** [comment, marked as answer]
For YouTube pop-up thing, you can hide the element (it's not blocking). And there's a transparent element behind it. That's why you can't click anything. You can hide that too. But still, when you open a video, it pauses it. If you continue to do so, it won't let you play.
If your problem is only on YouTube, [this is](https://www.reddit.com/r/uBlockOrigin/about/sticky?num=2) what you're looking for. If not, previous answer should be enough for you.

**Reply by Iftakharpy** [reply]
Thank you for the answer. This issue was only with YouTube. I wasn't able to interact with the YouTube page when I blocked all the DOM elements that were either blocking the view or mouse clicks. However, scrolling wouldn't work when the pop-up was blocked using uBO, and YouTube used to reset the scroll position. But manually closing the element wouldn't cause that to happen.

I am not sure what made them allow ad blockers now. I don't see any pop-ups, nor do I get any interruptions for using ad blockers these days.
